src/platform-includes/session-replay/setup/javascript.mdx

```diff
@@ -4,8 +4,6 @@ To set up the integration, add the following to your Sentry initialization. Seve
 // import Sentry from your framework SDK (e.g. @sentry/react) instead of @sentry/browser
 import * as Sentry from "@sentry/browser";
```

Comment:
We should make this platform-specific some time in the future, to show the correct package for each framework.